src/main/java/com/mraof/minestuck/MinestuckConfig.java

```diff
@@ -145,6 +146,8 @@ private Server(Builder builder)
 			lotusRestorationTime = builder.comment("Determines how many seconds it takes for the lotus blossom to regrow after the opening process has started.")
 					.defineInRange("lotusRestorationTime", 600, 30, Integer.MAX_VALUE);
 			hardMode = builder.define("hardMode", false);
+			wirelessBlocksTickRate = builder.comment("Determines the speed at which remote/wireless redstone blocks send/receive updates.")
+					.defineInRange("wirelessBlocksTickRate", 6, 1, 200);
```

Comment:
Agreed, and it may even cause some issues with certain puzzles. However I wanted to give people options since regular tile entity updates could cause lag, especially in a server setting

Reply:
After thinking about it more, it may cause inconsistencies in puzzles if the value were increased too much. Removing it